### PR TITLE
Correct inset styles of filled components on focus

### PR DESCRIPTION
**Pull request by SaiSan-WMDE**
💥 Breaking changes 💥

This applies the right box-shadow (and border) styles to filled elements on focus. Alias tokens were deleted or modified.

Affected components: 

- Primary buttons (progressive & destructive, iconOnly)
- Checkbox
- Toggle button

Issue: [T272626](https://phabricator.wikimedia.org/T272626)

### CODE REVIEW

**Review by micgro42 — APPROVED** [review]
Looks better and also seems fine from an engineering perspective :+1:

**Review by tzhelyazkova — APPROVED** [review]
LGTM too 👍 
I've seen these types of PRs mentioning breaking changes a few times.
Maybe the time has come to revisit the decision that all packages part of wikit will keep the same version number because while these changes are breaking for the tokens package, they are not breaking for the Vue components package.

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ development)_
 
 - _**(Breaking)**_ Width property is removed from all components
 - Cursor tokens
+- The token "box-shadow-button-primary"
```
Comment (SaiSan-WMDE) [inline CHANGELOG.md:55]:
```suggestion
- The token "box-shadow-inset-focus-button-primary" 
```

**Review by tzhelyazkova — APPROVED** [review]
Approving again to make the merge button active.

**Pull request merged**